webapp/channels/src/components/admin_console/localization/auto_translation_info.scss

```diff
@@ -0,0 +1,3 @@
+.autoTranslationInfo .sectionNoticeTitle {
+    margin-bottom: 0 !important;
```

Comment:
Why is this `!important`? Seeing this makes me think we're either reusing some CSS somewhere that we shouldn't be or that we're introducing two different slight variations on the same system console section component

Reply:
Well yes, there is somewhere an admin_console h4 class that is adding a margin-bottom that we don't want here and I did not want to disrupt other parts of the UI by overriding that, so I'm telling the component to just override it in the component itself. And this is again something that goes beyond the defined UI/UX that we have in place for the rest of the admin console.

Reply:
@matthewbirtch here too

Reply:
Let's remove the `!important` then. This should be more specific already because it uses a class name instead of an element name. We can address remove the more general style at some point if we decide we need to down the line